docs/config.schema.json

```diff
@@ -656,28 +656,16 @@
       "description": "The secrets section configures secrets used for encryption and signing of several systems. All secrets can be rotated, for more information on this topic go to: https://www.ory.sh/docs/hydra/advanced#rotation-of-hmac-token-signing-and-database-and-cookie-encryption-keys",
       "properties": {
         "system": {
-          "type": "array",
           "description": "The system secret must be at least 16 characters long. If none is provided, one will be generated. They key is used to encrypt sensitive data using AES-GCM (256 bit) and validate HMAC signatures. The first item in the list is used for signing and encryption. The whole list is used for verifying signatures and decryption.",
-          "items": {
-            "type": "string",
-            "minLength": 16
-          },
-          "minItems": 1,
-          "examples": [
-            [
-              "this-is-the-primary-secret",
-              "this-is-an-old-secret",
-              "this-is-another-old-secret"
-            ]
-          ]
-        },
-        "cookie": {
+          "default": [
+            "I AM NOT SECRET PLEASE REPLACE ME!!!"
```

Comment:
Please remove this immediately. This will cause any malformed config or inexperienced user to have this as their secret which is really bad. Never set hardcoded default secrets ever anywhere!